components/Navigation/Home.tsx

```diff
@@ -1,17 +1,62 @@
-import { View, Text } from "react-native";
```

Comment:
This could have been a simple default file just for the sake of pushing navigation (i.e. default App.tsx) and then doing this as separate PR, alas, for the sake of completing the project, it is fine.